crates/bevy_render/src/mesh/shape/regular_polygon.rs

```diff
@@ -105,3 +105,62 @@ impl From<Circle> for Mesh {
         Mesh::from(RegularPolygon::from(circle))
     }
 }
+
+#[cfg(test)]
+mod tests {
+    use crate::mesh::shape::RegularPolygon;
+    use crate::mesh::{Mesh, VertexAttributeValues};
+
+    fn fix_floats<const N: usize>(points: &mut [[f32; N]]) {
```

Comment:
This function name isn't super clear to me. Can you try to improve it or add some doc comments?

Reply:
Done.